### PR TITLE
add alternative type to response

### LINKED ISSUE #1011: show infos around identity vs. person first
* add person/identity first to the API response
* frontend link to an article

### PR DESCRIPTION
**Pull request by lsmith77**
fixes https://github.com/witty-works/nlp_api/issues/1011

* [x] bump version
* [ ] add German URL => https://github.com/witty-works/nlp_api/issues/1089

### CODE REVIEW

File: tests/test_general_cases/test_api-client/output.json [hunk 1]
```diff
@@ -5,19 +5,25 @@
         {
             "alternatives": [
                 {
-                    "context": "💡",
+                    "context": "💡 Person zuerst",
                     "inspiration": true,
-                    "text": "jemand, der erfahren und fachkundig ist"
+                    "text": "jemand, der erfahren und fachkundig ist",
+                    "type": "person_first",
```
Comment (lsmith77) [inline tests/test_general_cases/test_api-client/output.json:11]: @hellasol will adding this property cause issues for the browser extension?

the plan is if the type is not null and not equal to default that we turn the alternative label into a clickable link.

File: app/models.py [hunk 2]
```diff
@@ -802,6 +826,12 @@ def clean_alternatives(
                     context=alternative.label,
                 )
 
+                if alternative.type != AlternativeType.DEFAULT:
+                    variation.type = alternative.type
+                    variation.url = (
+                        "https://../en" if lang.lang == LangType.EN else "https://../de"
```
Comment (lsmith77) [inline app/models.py]: URLs TBD

File: tests/test_general_cases/test_api-client/output.json (hunk 1 above)
Comment (lsmith77) [reply tests/test_general_cases/test_api-client/output.json:11]: yes .. so version needs to be increased

File: app/models.py [hunk 3]
```diff
@@ -827,6 +851,14 @@ def clean_alternatives(
                     context=alternative.label,
                 )
 
+                if alternative.type != AlternativeType.DEFAULT:
+                    variation.type = alternative.type
+                    variation.url = (
+                        "https://www.witty.works/en/blog/person-first-vs.-identity-first-understanding-the-approaches"
+                        if lang.lang == LangType.EN
+                        else "https://www.witty.works/en/blog/person-first-vs.-identity-first-understanding-the-approaches"
```
Comment (lsmith77) [inline app/models.py:859]: add german blog URL

**Review by hellasol — APPROVED** [review]
remember: add german blog URL and bump version 👍

**Pull request merged**